src/BookingForm.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from "react";
+
+function BookingForm(props) {
+  const [title, setTitle] = useState("");
+  const [firstName, setFirstName] = useState("");
+  const [surname, setSurname] = useState("");
+  const [roomId, setRoomId] = useState("");
+  const [checkInDate, setCheckInDate] = useState("");
+  const [checkOutDate, setCheckOutDate] = useState("");
+  const [email, setEmail] = useState("");
+
+  function handleChange(event) {
+    if (event.target.name === "title") {
+      console.log(event.target.value);
+      setTitle(event.target.value);
+    } else if (event.target.name === "firstName") {
+      setFirstName(event.target.value);
+    } else if (event.target.name === "surname") {
+      setSurname(event.target.value);
+    } else if (event.target.name === "email") {
+      setEmail(event.target.value);
+    } else if (event.target.name === "roomId") {
+      setRoomId(event.target.value);
+    } else if (event.target.name === "checkInDate") {
+      setCheckInDate(event.target.value);
+    } else if (event.target.name === "checkOutDate") {
+      setCheckOutDate(event.target.value);
+    }
+  }
```

Comment:
There is a way we can change our state here so we don't have all the `else if` statements, can you see what it is? 😄 
Hint: It would involve just having one `useState`